src/LoadingShim/index.js

```diff
@@ -36,6 +49,8 @@ LoadingShim.propTypes = {
   isLoading: PropTypes.bool,
   /** Optional value for `data-testid` attribute */
   testId: PropTypes.string,
+  /** Size of the loading indicator */
+  size: PropTypes.oneOf(["small", "large"]),
```

Comment:
for consistency with other size names:

```suggestion
  size: PropTypes.oneOf(["s", "l"]),
```